### PR TITLE
feat: dbt 1.6

### PR DESCRIPTION
**Pull request by ZeRego**
Closes: #6925

### Description:

Changes:
- update docker to install dbt 1.6
- add support for dbt 1.6 + manifest v10 when compiling project via CLI or the APP
- this PR doesn't provide support for metricflow so we are ignore the metrics from the manifest
- update cli tests

### CODE REVIEW

File: .github/workflows/cli-tests.yml [hunk 1]
```diff
@@ -35,9 +35,15 @@ jobs:
       ## Install pip
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.7.x'
+          python-version: '3.9.x'
```
Comment (ZeRego) [inline .github/workflows/cli-tests.yml:38]: dbt 1.6 dropped support for python 3.7

Comment (owlas) [reply .github/workflows/cli-tests.yml:38]: do you know what we're running in the container? It looks like we're not pinning just taking the bundled debian version

File: packages/common/src/dbt/schemas/manifestV10.json
Comment (owlas) [inline packages/common/src/dbt/schemas/manifestV10.json:1]: What's your process for updating the manifest @ZeRego ?

File: packages/cli/src/handlers/compile.ts [hunk 2]
```diff
@@ -138,7 +139,9 @@ ${errors.join('')}`),
         validModelsWithTypes,
         false,
         manifest.metadata.adapter_type,
-        Object.values(manifest.metrics),
+        (await getDbtManifest()) === DbtManifestVersion.V10
```
Comment (owlas) [inline packages/cli/src/handlers/compile.ts:142]: nice!

**Review by owlas — APPROVED** [review]

**Review by rephus — APPROVED** [review]
LGTM

File: .github/workflows/cli-tests.yml (hunk 1 above)
Comment (ZeRego) [reply .github/workflows/cli-tests.yml:38]: Python 3.9.2

File: packages/common/src/dbt/schemas/manifestV10.json
Comment (ZeRego) [reply packages/common/src/dbt/schemas/manifestV10.json:1]: Manual process at the moment. I go to https://schemas.getdbt.com/dbt/manifest/v10.json and copy paste the contents. I was wondering if it possible to just pass the url to AVJ so it's always up to date.

**Pull request merged**